index.html

```diff
@@ -0,0 +1,194 @@
+          <small><strong>Cost to Adopt:</strong> $123.45</small>
+          <figcaption>Consectetur tempora magnam optio quibusdam! Lorem ipsum dolor sit amet, consectetur adipisicing
+            elit.
+          </figcaption>
+        </figure>
+        <a href="#" onclick="addToTotal(123.45)" class="btn btn-adopt">Adopt</a>
+      </div>
+
+      <div class="card">
+        <figure>
+          <img onclick="alertDogInfo('Jack', 'Beagle', 123.45)" src="images/jack-card.jpg" alt="Staring sad dog">
+          <h4>Jack</h4>
+          <small><strong>Cost to Adopt:</strong> $123.45</small>
+          <figcaption>Consectetur tempora magnam optio quibusdam! Lorem ipsum dolor sit amet, consectetur adipisicing
+            elit.
+          </figcaption>
+        </figure>
+        <a href="#" onclick="addToTotal(123.45)" class="btn btn-adopt">Adopt</a>
+      </div>
+
+      <div class="card">
+        <figure>
+          <img onclick="alertDogInfo('Duffy', 'Pitbull', 123.45)" src="images/duffy-card.jpg" alt="Staring sad dog">
+          <h4>Duffy</h4>
+          <small><strong>Cost to Adopt:</strong> $123.45</small>
+          <figcaption>Consectetur tempora magnam optio quibusdam! Lorem ipsum dolor sit amet, consectetur adipisicing
+            elit.
+          </figcaption>
+        </figure>
+        <a href="#" onclick="addToTotal(123.45)" class="btn btn-adopt">Adopt</a>
+      </div>
+
+      <div class="card">
+        <figure>
+          <img onclick="alertDogInfo('Lucas', 'Shepherd', 123.45)" src="images/lucas-card.jpg" alt="Staring sad dog">
+          <h4>Lucas</h4>
+          <small><strong>Cost to Adopt:</strong> $123.45</small>
+          <figcaption>Consectetur tempora magnam optio quibusdam! Lorem ipsum dolor sit amet, consectetur adipisicing
+            elit.
+          </figcaption>
+        </figure>
+        <a href="#" onclick="addToTotal(123.45)" class="btn btn-adopt">Adopt</a>
+      </div>
+
+      <div class="card">
+        <figure>
+          <img onclick="alertDogInfo('Jake', 'Labrador', 123.45)" src="images/jake-card.jpg" alt="Staring sad dog">
+          <h4>Jake</h4>
+          <small><strong>Cost to Adopt:</strong> $123.45</small>
+          <figcaption>Consectetur tempora magnam optio quibusdam! Lorem ipsum dolor sit amet, consectetur adipisicing
+            elit.
+          </figcaption>
+        </figure>
+        <a href="#" onclick="addToTotal(123.45)" class="btn btn-adopt">Adopt</a>
+      </div>
+
+      <div class="card">
+        <figure>
+          <img onclick="alertDogInfo('Angus', 'Boxer', 123.45)" src="images/angus-card.jpg" alt="Staring sad dog">
+          <h4>Angus</h4>
+          <small><strong>Cost to Adopt:</strong> $123.45</small>
+          <figcaption>Consectetur tempora magnam optio quibusdam! Lorem ipsum dolor sit amet, consectetur adipisicing
+            elit.
+          </figcaption>
+        </figure>
+        <a href="#" onclick="addToTotal(123.45)" class="btn btn-adopt">Adopt</a>
```

Comment:
I would recommend using a button element instead of anchor so the href="#" does not force the window to the top of the page on each click.

Reply:
Great advice and updated. Thanks.